mathlibtools/lib.py

```diff
@@ -148,29 +148,21 @@ def get_mathlib_archive(rev: str, url:str = '', force: bool = False,
     Return the archive Path. Will raise LeanDownloadError if nothing works.
     """
 
-    # we check for xz archives first
-    fnames = [rev + '.tar.xz', rev + '.tar.gz', rev + '.tar.bz2']
-    paths = [DOT_MATHLIB/fname for fname in fnames]
+    fname = rev + '.tar.xz'
+    path = DOT_MATHLIB/fname
     if not force:
         log.info('Looking for local mathlib oleans')
-        for path in paths:
-            if path.exists():
-                log.info('Found local mathlib oleans')
-                return path
-    log.info('Looking for remote mathlib oleans')
-    for fname, path in zip(fnames, paths):
-        try:
-            base_url = url or get_download_url()
-            download(base_url+fname, path)
-            log.info('Found mathlib oleans at '+base_url)
+        if path.exists():
+            log.info('Found local mathlib oleans')
             return path
-        except LeanDownloadError:
-            pass
-    log.info('Looking for GitHub mathlib oleans')
-    # nightlies will only store gz archives
-    path = DOT_MATHLIB / (rev + '.tar.gz')
-    download(nightly_url(rev, repo), path)
-    log.info('Found GitHub mathlib oleans')
+    log.info('Looking for remote mathlib oleans')
+    try:
+        base_url = url or get_download_url()
+        download(base_url+fname, path)
+        log.info('Found mathlib oleans at '+base_url)
+        return path
+    except LeanDownloadError:
+        pass
     return path
```

Comment:
```suggestion
    base_url = url or get_download_url()
    download(base_url+fname, path)
    log.info('Found mathlib oleans at '+base_url)
    return path
```
since the exception will be caught and handled at https://github.com/leanprover-community/mathlib-tools/blob/78c58aa51445e2833a69d09d1adecc8699a29539/mathlibtools/leanproject.py#L218